concourse/templates.libsonnet

```diff
@@ -329,7 +329,7 @@
           [if vault then 'vault_token_file']: 'vault/token',
           [if vault && vault_secrets != null then 'vault_secrets']: secret_array,
           [if vault && vault_configs != null then 'vault_configs']: config_array,
-          [if source != null && manifests != null then 'manifest_path']: source + '/' + manifests,
```

Comment:
This changes the behavior and will break any existing pipelines using this library

Reply:
ya, I just noticed